src/analysis/specific_limits/basic.lean

```diff
@@ -41,6 +42,34 @@ lemma tendsto_one_div_add_at_top_nhds_0_nat :
 suffices tendsto (λ n : ℕ, 1 / (↑(n + 1) : ℝ)) at_top (𝓝 0), by simpa,
 (tendsto_add_at_top_iff_nat 1).2 (tendsto_const_div_at_top_nhds_0_nat 1)
 
+/-- The limit of `n / (n + x)` is 1, for any constant `x` (valid in `ℝ` or any topological division
+algebra over `ℝ`, e.g., `ℂ`).
+
+TODO: introduce a typeclass saying that `1 / n` tends to 0 at top, making it possible to get this
+statement simultaneously on `ℚ`, `ℝ` and `ℂ`. -/
+lemma tendsto_coe_nat_div_add_at_top
+  {𝕜 : Type} [division_ring 𝕜] [topological_space 𝕜] [char_zero 𝕜] [algebra ℝ 𝕜]
+  [has_continuous_smul ℝ 𝕜] [has_continuous_inv₀ 𝕜] [topological_ring 𝕜]
```

Comment:
```suggestion
  [has_continuous_smul ℝ 𝕜] [topological_division_ring 𝕜]
```

Reply:
Why? This is strictly less general, because it imposes continuity of addition as well, which is not needed for the proof. I doubt the extra generality is ever likely to be relevant; but still, why ask for more than we need?

Reply:
(oops, posted some nonsense just then, please ignore)

Reply:
I remember now why I wrote it this way. For some reason the typeclass machinery isn't able to deduce `[topological_division_ring ℂ]`, so if you write the lemma this way, you have to repeat `haveI : topological_division_ring ℂ, from topological_division_ring.mk` every time you want to use it. This works, but it's ugly; what's the correct solution?

Reply:
I think we're missing a `normed_division_ring.to_topological_division_ring` instance, which can probably be proved as `{}`

Reply:
Yes, that works! I've pushed the fix.